File: Data Structures & Algorithms/concatenation-of-array/submission-1.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int[] res = new int[nums.Length * 2];
        for(int i = 0; i< nums.Length; i++){
            res[i] = nums[i];
        }
        for(int i = 0; i< nums.Length; i++){
            res[nums.Length + i] = nums[i];
        }
        return res;
    }
}

File: Data Structures & Algorithms/number-of-good-pairs/submission-1.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int arr[101] ={0};
        for (int i : nums)
        {
            arr[i]++;
        }

        int res = 0;
        for (int i : arr)
        {
            res += cal(i);
        }

        return res;
    }

    int cal(int n)
    {
        int sum = 0;
        for (int i = 1;i < n; i++)
        {
            sum += i;
        }
        cout<<n <<" "<< sum<< endl;

        return sum;
    }
};